docs/tutorials/getting-started.md

```diff
@@ -1,92 +0,0 @@
-# Getting Started
```

Comment:
i think this page is old. it is not accessible in gitbook because it is not in `SUMMARY.md`. we have a separate `getting-started.md` that is linked